Examples/src/CheckCollection.cpp

```diff
@@ -0,0 +1,51 @@
+#include "GaudiAlg/GaudiAlgorithm.h"
+
+#include "FWCore/DataHandle.h"
+
+#include "datamodel/MCParticleCollection.h"
+#include "datamodel/TrackHitCollection.h"
+#include "datamodel/GenVertex.h"
+
+
+class CheckCollection : public GaudiAlgorithm {
```

Comment:
CheckCollection sounds too generic to me. Can you use a name like FCCTestConsumer or something like that?